model/waterfall.go

```diff
@@ -103,7 +103,8 @@ func GetActiveWaterfallVersions(ctx context.Context, projectId string, opts Wate
 
 // GetAllWaterfallVersions returns all of a project's versions within an inclusive range of orders.
 func GetAllWaterfallVersions(ctx context.Context, projectId string, minOrder int, maxOrder int) ([]Version, error) {
```

Comment:
I think there might be a small bug with the revision order number range, when I load the default Spruce waterfall I see 13 inactive versions at the end (which doesn't show up on the `main` branch). Comparing with the Project Health page, I think these commits might actually be active so they wouldn't end up being collapsed 👀

<img width="464" alt="Screenshot 2024-10-03 at 11 35 22 AM" src="https://github.com/user-attachments/assets/1f4fb724-aa33-4fed-8972-2a0b7b3bc3ac">